library/src/camera_impl.cxx

```diff
@@ -202,7 +209,23 @@ camera& camera_impl::resetToDefault()
 //----------------------------------------------------------------------------
 camera& camera_impl::resetToBounds()
 {
+
+#if VTK_VERSION_NUMBER < VTK_VERSION_CHECK(9, 0, 20210331)
   this->Internals->VTKRenderer->ResetCamera();
+#else
+#if F3D_MODULE_EXTERNAL_RENDERING
+  if (vtkExternalOpenGLRenderWindow::SafeDownCast(
```

Comment:
Calling `IsA` might be better, what do you think?

Reply:
definitely